actor/kvcounter/README.md

```diff
@@ -0,0 +1,17 @@
+# K/V Counter
+
+This actor accepts `GET` requests and for each URL it gets, 
+it will increment a call counter for that URL and return the result 
```

Comment:
I think this may read better as 
```
... it will increment counter based on that URL path and ...
```

Reply:
It's a nitpick for sure, but I think it makes it a bit clearer what the actor is doing